index.php

```diff
@@ -14,6 +14,17 @@
 <div class="twrapper">
 <table class="people">
 	<thead>
+		<tr>
+		    	<th colspan="3" style="border-bottom: 1.5px solid #4F5B93; text-align: center !important;">
+				<?php if ($page > 1): ?>
+		    		<a class="pagination prev" href="?page=<?php echo $page-1?>">« Previous page</a>
+				<?php endif ?>
+		    		<span class="page"><?php echo $page ?></span>
+				<?php if ($x == 49): ?>
```

Comment:
The variable `$x` is not yet defined here. Also, it will only be `49` after the `foreach` loop (when looping over a full "page" of users).